src/modules/submodules/consensus.py

```diff
@@ -102,7 +115,7 @@ def _get_member_info(self, blockstamp: BlockStamp) -> MemberInfo:
 
         return MemberInfo(
             is_member=is_member,
-            last_report_ref_slot=last_report_ref_slot,
+            last_report_ref_slot=last_member_report_ref_slot,
```

Comment:
Let's rename last_report_ref_slot -> last_member_report_ref_slot in MemberInfo

Reply:
done